### DISCUSSION TITLE
Image Analyze can't work with batch? And how to output numbers instead of graph?

### DISCUSSION THREAD

**Discussion by le-khang** [WASasquatch/was-node-suite-comfyui#241] [General]
Hi WASasquatch,
I like your Image Analyze node since I don't have to export image then go to Photopea/Photoshop to check its data.
However when I generate 10 images it shows this error below.
And can it output level data as numbers please? (Such as mean value, standard deviation, etc.)

![2023-10-22 13_39_57-ComfyUI](https://github.com/WASasquatch/was-node-suite-comfyui/assets/35536989/60ee3561-57e4-43c2-99a9-5a1d69f8510e)

**Comment by WASasquatch** [comment]
How would you differentiate or handle the output? Additionally I am using BLIP 1, the model just returns the caption. I am looking to update to BLIP2 when I have more time.
